spec_driven_model/models/spec_import.py

```diff
@@ -228,7 +211,24 @@ def _prepare_import_dict(self, vals, model=None):
                 vals[related_m2o] = comodel.match_or_create_m2o(sub_val, vals)
             else:  # search res.country with Brasil for instance
                 vals[related_m2o] = model.match_or_create_m2o(sub_val, vals, comodel)
-        return vals
+
+        defaults = model.with_context(
```

Comment:
A questão do valor default do campo nfe40_modFrete estar sobrescrevendo o valor da nota continua acontecendo. O que acontece é que o método _prepare_import_dict é chamado diversas vezes para o modelo l10n_br_fiscal.document.
    Quando ele é chamado para as tags dentro do campo transp o valor do campo modFrete na nota é respeitado. O problema é que o método é chamado novamente depois, em um momento onde o modFrete não está em vals, e isso está causando a sobrescrita do campo.
    Acredito que essa chamada ao default_get precisa ir pra outro lugar. Talvez para o método build_attrs, ou de volta para o build_from_biding, porém após a chamada do build_attrs.
   Fora isso, não encontrei mais nenhum problema

Reply:
Olá @hirokibastos, obrigado pelo aviso. Vc teria como compartilhar o XML e /ou pedaço de código relevante que vc tá usando? O XML vc poderia tirar o que tiver dentro do dest e do emit se for dados de prod. O melhor mesmo seria um teste alterado que mostra o problema, mesmo se depender de algum código temporário a mais. Se tiver como passar uma branch com isso me ajudaria bastante...

Reply:
Nos testes atuais, todos os xml estão apenas com a tag modFrete=9 dentro da tag transp. Para ver o erro, basta adicionar informação dentro desta tag. 

Segue um exemplo que criei usando um gerador de cnpj e ie:

<transp>
<modFrete>3</modFrete>
<transporta>
<CNPJ>91853680000141</CNPJ>
<xNome>Camila e Daniela Entulhos Ltda</xNome>
<IE>495903034946</IE>
<xEnder>Rua São Bartolomeu 849</xEnder>
<xMun>SALTO</xMun>
<UF>SP</UF>
</transporta>
<vol>
<qVol>36</qVol>
<esp>AMARRADOS</esp>
<pesoL>1078.000</pesoL>
<pesoB>1078.000</pesoB>
</vol>
</transp>

Reply:
valeu. Vou olhar logo. Se eu não conseguir resolver facilemente eu vou cortar o PR em 2 para a gente poder fazer o merge do fix do StackedModel antes da melhoria do dry run.